.github/workflows/ci.yaml

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
Please rebase your PR on the latest master. It should have `.github/workflows/ci.yaml`